candle-transformers/src/models/mistral.rs

```diff
@@ -37,6 +37,22 @@ impl Config {
             use_flash_attn,
         }
     }
+    pub fn config_chat_ml(use_flash_attn: bool) -> Self {
+        Self {
```

Comment:
Could you add a pointer to the config/model on the huggingface hub? This will make it easier to be sure what this config should be used for.

Reply:
I added a link to the config on the [model README.md](https://huggingface.co/DanielClough/Candle_Mistral-7B-OpenOrca/blob/main/README.md).